crates/lumina_client/src/ui/game_ui/settings.rs

```diff
@@ -0,0 +1,98 @@
+use bevy::prelude::*;
+use bevy_kira_audio::prelude::*;
+use lumina_ui::prelude::*;
+use velyst::prelude::*;
+
+pub(super) struct SettingsUiPlugin;
+
+use crate::audio::{Background, SoundFx};
+#[derive(Resource)]
+pub struct AudioSettings {
+    pub bgm_volume: f64,
+    pub vfx_volume: f64,
+}
+
+impl Default for AudioSettings {
+    fn default() -> Self {
+        AudioSettings {
+            bgm_volume: 0.5,
+            vfx_volume: 0.5,
+        }
+    }
+}
```

Comment:
Nit pick: this project normally have struct definitions added below the file (apart from Plugin structs)